addons/web/static/src/core/datetime/datetime_picker.xml

```diff
@@ -66,6 +66,13 @@
                                 <t t-set="unitIndex" t-value="1" />
                                 <t t-set="unitList" t-value="availableMinutes" />
                             </t>
+                            <t t-if="availableSeconds.length">
+                                <span>:</span>
+                                <t t-call="web.DateTimePicker.Select">
+                                    <t t-set="unitIndex" t-value="2" />
+                                    <t t-set="unitList" t-value="availableSeconds" />
+                                </t>
+                            </t>
```

Comment:
In website, the countdown snippet shows seconds, and in order for the countdown to showcase how it looks upon insertion, it will pick a date in the future that will have non-zero seconds. If the seconds are not displayed in the picker, you can't reset them to 0.

Reply:
add a test?